### ISSUE TITLE
What do CT measure- real or apparent current?

### ISSUE THREAD

**Issue by nipunbatra** [nipunbatra/Home_Deployment#51]

**Comment by amarjeet-iiitd** [comment]
There is nothing like real or apparent current. It measures current.

Best regards
Amarjeet Singh

**Comment by nipunbatra** [comment]
Yes, that was a stupid mistake from my side.

Current is V/Z or V/R?

Let me rephrase: If i multiply instantaneous voltage (obtained from smart
meter) with current- what do we get- Real Power or Apparent Power? This
depends on previous answer.

**Comment by amarjeet-iiitd** [comment]
You will get apparent power.

Apparent power - v_i
Real power - v_i*cos theta

Best regards
Amarjeet Singh

**Comment by nipunbatra** [comment]
Ok. Thanks.
And since we dont measure phase per circuit, so we cant get real power.
